test/harness/acceptance_network.go

```diff
@@ -31,7 +31,7 @@ func NewAcceptanceTestNetwork(numNodes uint32, logFilters []log.Filter, consensu
 		WithOutput(log.NewOutput(output).
 		WithFormatter(log.NewHumanReadableFormatter())).
 		WithFilters(logFilters...).
-		WithFilters(log.OnlyErrors())
+		WithFilters(log.Or(log.OnlyErrors(), log.OnlyCheckpoints()))
```

Comment:
I'm not sure this is a good enough approach. What if the `logFilters` param above has some rejecting filters? it will make the `Or` filter irrelevant. We can keep this for the mean time, but note that the entire filter concept might require some overhaul